src/features/query-bar/validator.js

```diff
@@ -1,15 +1,19 @@
 // @flow
+import { isNaN } from 'lodash';
+
 import { FLOAT } from './constants';
 
-const floatRegex = /[^$,.\d]/;
+const isFloat = (value: string) => {
+    return !/^\s*$/.test(value) && !isNaN(Number(value));
```

Comment:
Is there a reason you aren't using the `parseFloat` function that's native to the browser?
Performance is probably similar but seems weird to implement your own regex.

Reply:
I can use parseFloat! I wasn't aware of that library method.

Reply:
Actually I can stick with my regex expression. It looks like when I use parseFloat('1.0.') this returns 1 when it should instead return NaN.

Reply:
It very much seems like `lodash/isFixed` should be a separate check for all of these validations rather than having them baked in.